lib/bdev/nvme/blockdev_nvme.c

```diff
@@ -506,16 +507,62 @@ probe_cb(void *cb_ctx, const struct spdk_nvme_transport_id *trid,
 }
 
 static void
+abort_cb(void *arg, const struct spdk_nvme_cpl *cpl)
+{
+	struct spdk_nvme_ctrlr *ctrlr = arg;
+	assert(ctrlr != NULL);
+	spdk_decr_abort_count(ctrlr);
+}
+
+static void
 blockdev_nvme_timeout_cb(struct spdk_nvme_ctrlr *ctrlr,
-			 struct spdk_nvme_qpair *qpair, void *cb_arg)
+			 void *cb_arg,
+			 uint16_t timeout_count,
+			 uint16_t cid,
+			 uint16_t sqid)
 {
 	int rc;
 
-	SPDK_WARNLOG("Warning: Detected a timeout. ctrlr=%p qpair=%p\n", ctrlr, qpair);
+	if (cb_arg) {
+		SPDK_WARNLOG("Warning: Detected a timeout. user_data=%p ctrlr=%p \n", cb_arg, ctrlr);
```

Comment:
cb_arg is not used by the NVMe blockdev, so I don't think there's any reason to change this printout.